packages/core/src/modules/x509/X509Certificate.ts

```diff
@@ -228,6 +311,11 @@ export class X509Certificate {
     return certificate.subject
   }
 
+  public get issuer() {
+    const certificate = new x509.X509Certificate(this.rawCertificate)
```

Comment:
Should we store an instance of the x509 lib cert in this class? I'm not sure on the overhead of parsing